### PR TITLE
fix(@desktop/communities): changing contact nickname

### LINKED ISSUE #3283: communities local usernames not being reflected
1. click on a profile, set their local username
2. check user list

result: they still have 3 word username
expected result: it should show their local username

### PR DESCRIPTION
**Pull request by pinebit**
Fixes #3283

### CODE REVIEW

**Review by alaibe — APPROVED** [review]

File: src/app/chat/views/community_members_list.nim [hunk 1]
```diff
@@ -114,17 +117,6 @@ QtObject:
       of CommunityMembersRoles.StatusType: result = newQVariant(self.memberStatus(communityMemberPubkey))
       of CommunityMembersRoles.Online: result = newQVariant(self.isOnline(communityMemberPubkey))
       of CommunityMembersRoles.SortKey: result = newQVariant(self.sortKey(communityMemberPubkey))
-      
-  proc rowData(self: CommunityMembersView, index: int, column: string): string {.slot.} =
```
Comment (alaibe) [inline src/app/chat/views/community_members_list.nim:118]: I think this is used to render suggestion (which is currently broken). So we should probably not delete that

Comment (alaibe) [reply src/app/chat/views/community_members_list.nim:118]: pls see: https://github.com/status-im/status-desktop/pull/3341

Comment (pinebit) [reply src/app/chat/views/community_members_list.nim:118]: I did not find any usage, but I trust your opinion and will bring it back. Thanks!

Comment (pinebit) [reply src/app/chat/views/community_members_list.nim:118]: Done.

**Pull request merged**